src/main/java/frc/robot/subsystems/shooter/FlywheelIOTalonFX.java

```diff
@@ -90,18 +91,15 @@ public FlywheelIOTalonFX() { // Object to set different flywheel configs
     voltageRequest = new VoltageOut(0.0);
   }
 
-  /**
-   * @param inputs has all the inputs, updates and logs them
-   */
   @Override
   public void updateInputs(
       FlywheelIOInputs inputs) { // gets current values for motors and puts them into a log
     BaseStatusSignal.refreshAll(
         leaderPosition, leaderVelocity, leaderAppliedVolts, leaderCurrent, followerCurrent);
     inputs.positionRotations =
         leaderPosition.getValueAsDouble()
-            / ShooterConstants.GEAR_RATIO; // converted to radians to gear ratio math
-    inputs.velocityRPM = (leaderVelocity.getValueAsDouble() * 60.0) / ShooterConstants.GEAR_RATIO;
+            / FlywheelConstants.GEAR_RATIO; // converted to radians to gear ratio math
+    inputs.velocityRPM = (leaderVelocity.getValueAsDouble() * 60.0) / FlywheelConstants.GEAR_RATIO;
```

Comment:
I think this * 60.0 should be a constant, you can put it in HardwareConstants or something

Reply:
ok